Objects/codeobject.c

```diff
@@ -1400,18 +1400,36 @@ _PyCode_GetVarnames(PyCodeObject *co)
     return get_localsplus_names(co, CO_FAST_LOCAL, co->co_nlocals);
 }
 
+PyObject *
+PyCode_GetVarnames(PyCodeObject *code)
+{
+    return _PyCode_GetVarnames(code);
```

Comment:
What's the advantage of separate wrapper functions rather than just removing the leading `_` from the wrapped functions?

Reply:
I concur wuth @gvanrossum, unless it's implemented as a static inline function, I see no advance, but it's not the case here. Just rename _PyCode_GetVarnames() to PyCode_GetVarnames(). Same remark for the 2 other functions.